content/post/70-js-ipfs-0-38.md

```diff
@@ -0,0 +1,118 @@
+---
+date: 2019-08-30
+url: 70-js-ipfs-0-38
+title: js-ipfs 0.38.0 released
+author: Alex Potsides
+---
+
+> Garbage collection, gossipsub, IPNS support for ipfs resolve, oh my!
+
+```console
+$ npm install -g ipfs
+```
+
+# 🔦 Highlights
+
+## 🗑️ Garbage collection
+
+Support for garbage collection has shipped with `0.38.0`! This means that blocks in your IPFS Repo will be removed when gc is run unless they are pinned.
```

Comment:
```suggestion
Support for garbage collection has shipped with `0.38.0`! This means that blocks in your IPFS Repo will be removed once gc runs, unless they are pinned.
```

Reply:
Not sure I prefer `once`.  Sounds so final, like it's not going to be run again.

Reply:
`whenever gc is run`?

Reply:
`Whensoever gc is run`?

Reply:
I kid! I made it `whenever`.

Reply:
sounds good